Sources/Client/Model/Pagination.swift

```diff
@@ -8,6 +8,22 @@
 
 import Foundation
 
+public typealias Pagination = Set<PaginationOption>
+
+public extension Set where Element == PaginationOption {
+    var limit: Int? {
+        first(where: { $0.limit != nil })?.limit
+    }
+    
+    var offset: Int? {
+        first(where: { $0.offset != nil })?.offset
+    }
+    
+    func encode(to encoder: Encoder) throws {
+        try forEach({ try $0.encode(to: encoder) })
+    }
```

Comment:
Why is that needed?

Reply:
Otherwise `Set` encodes as `unkeyedContainer` which results in a runtime failure. We need it to encode in this format.